src/coreclr/scripts/superpmi.py

```diff
@@ -1711,8 +1711,13 @@ async def create_one_artifact(jit_path: str, location: str, flags) -> str:
                                 logging.debug("%sInvoking: %s", print_prefix, " ".join(command))
                                 proc = await asyncio.create_subprocess_shell(" ".join(command), stderr=asyncio.subprocess.PIPE, env=modified_env)
                                 await proc.communicate()
```

Comment:
I think you need to delete the existing `await proc.communicate()` line.

Reply:
```suggestion
```

Reply:
Oops, indeed it was leftover, though when I tested it it still seemed to work